src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -26,23 +26,51 @@ import Details from '@common/components/Details';
 import PageSearchForm from '@common/components/PageSearchForm';
 import RelatedAside from '@common/components/RelatedAside';
 import ReleaseDataAndFilesAccordion from '@common/modules/release/components/ReleaseDataAndFilesAccordion';
-import React, { useCallback, useMemo } from 'react';
-import { generatePath, useLocation } from 'react-router';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { generatePath, Prompt, useLocation } from 'react-router';
 
 interface MethodologyLink {
   key: string;
   title: string;
   url: string;
 }
 
+const handleBeforeUnload = (showPrompt: boolean) => {
+  window.onbeforeunload = event => {
```

Comment:
Would suggest using the `addEventListener` syntax to be consistent with how we usually define event listeners on the DOM i.e. 
```ts
window.addEventListener('beforeunload', event => ...)
```

Reply:
Done